mysql-test/suite/binlog/t/percona_binlog_snapshot_gtid_executed-master.opt

```diff
@@ -0,0 +1 @@
+--enforce-gtid-consistency --gtid-mode=ON
```

Comment:
In 8.0 this test should be in `binlog_gtid` test suite.
This .opt file can be removed then.

Reply:
done